src/client.rs

```diff
@@ -64,6 +66,16 @@ fn handle_list_disks(s: &mut Socket) {
     };
 }
 
+fn handle_jira_tickets(s: &mut Socket) -> BynarResult<()>{
+    trace!("start of test get_jir_tickets");
```

Comment:
I think the text here would be better as: `start of handle_jira_tickets` or `handle_jira_tickets called` so we know where in the code we are.

Reply:
Fixed